brew_view/static/src/js/services/system_service.js

```diff
@@ -32,8 +32,16 @@ export default function systemService($http) {
     return $http.delete('api/v1/systems/' + system.id);
   };
 
-  SystemService.getSystems = function() {
-    return $http.get('api/v1/systems');
+  SystemService.getSystems = function(includeCommands, includeFields, excludeFields) {
```

Comment:
It seems like here we have the potential to set params/headers to undefined/null. I think we want to minimize the number of things we send that aren't useful. Do you think it's important enough to change it?

Reply:
Setting them to undefined is how we prevent them from being sent. If you only want to exclude fields you'd call `(undefined, undefined, 'commands')` and that wouldn't send any query parameters or the `include_fields` header.